### PR TITLE
 jenkins 2.89 & jenkins-lts 2.73.3

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

https://jenkins.io/security/advisory/2017-11-08/

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/jenkins-lts.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+  sha256 "fcfb932a84ce03795abbf51ae7799439278155b0d42242d1d79a187034649fc8"
 
   bottle :unneeded
 
   depends_on :java => "1.7+"
 
   def install
     system "jar", "xvf", "jenkins.war"
-    libexec.install Dir["jenkins.war", "WEB-INF/jenkins-cli.jar"]
+    libexec.install %w[jenkins.war WEB-INF/jenkins-cli.jar]
```
Comment (ilovezfs) [inline Formula/jenkins-lts.rb]: The correct fix here is `libexec.install "jenkins.war", "WEB-INF/jenkins-cli.jar"`

Comment (DomT4) [reply Formula/jenkins-lts.rb]: Sure, will change. Wasn't sure which format was the most desired here these days to be honest.

**Comment by DomT4** [comment]
Should be okay now, once CI is content again.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
:tada:

**Comment by DomT4** [comment]
We got there in the end 🙈. Thanks @ilovezfs.

**Pull request merged**